sdk/src/main/java/com/qonversion/android/sdk/internal/user/storage/UserDataStorageImpl.kt

```diff
@@ -0,0 +1,41 @@
+package com.qonversion.android.sdk.internal.user.storage
+
+import androidx.annotation.VisibleForTesting
+import com.qonversion.android.sdk.internal.common.StorageConstants
+import com.qonversion.android.sdk.internal.common.localStorage.LocalStorage
+import com.qonversion.android.sdk.internal.exception.ErrorCode
+import com.qonversion.android.sdk.internal.exception.QonversionException
+
+internal class UserDataStorageImpl(
+    private val localStorage: LocalStorage,
+) : UserDataStorage {
+
+    @VisibleForTesting
+    var originalId: String? = localStorage.getString(StorageConstants.OriginalUserId.key)
+
+    @VisibleForTesting
+    var identityId: String? = localStorage.getString(StorageConstants.UserId.key)
```

Comment:
Let's use `StorageConstants.IdentityUserId.key`

Reply:
My vote is also for `StorageConstants.IdentityUserId.key`